domain/src/main/java/org/oppia/android/domain/devoptions/ForceDownloadRemoteParametersController.kt

```diff
@@ -0,0 +1,22 @@
+package org.oppia.android.domain.devoptions
+
+import org.oppia.android.domain.platformparameter.PlatformParameterControllerDebugImpl
+import org.oppia.android.util.data.DataProvider
+import javax.inject.Inject
+import javax.inject.Singleton
+
+/** Controller for force-downloading platform parameters. */
```

Comment:
Here and all KDocs below
```suggestion
/** Controller for force-downloading remote platform parameters and feature flags. */
```

Reply:
updated